src/utils/asyncPopulate.js

```diff
@@ -12,7 +12,7 @@ export default function asyncPopulate(target, source) {
     if (Array.isArray(source[attr])) {
       target[attr] = [];
       promise = asyncPopulate(target[attr], source[attr]);
-    } else if (source[attr] === null) {
+    } else if (!source[attr]) {
       target[attr] = null;
```

Comment:
This should be:

```js
    } else if (source[attr] === null || source[attr] === undefined) {
      target[attr] = source[attr];
    }
```

Reply:
@aexmachina you're right! By coercing to boolean all "falsy" values will be included in the statement, so I'll change it to the code you suggest. Thanks! 👍

Reply:
That's still not right. We need to preserve the `undefined`, so should be `target[attr] = source[attr]`

Reply:
@aexmachina I agree, if we need to preserve the `undefined` then this is the way to go. I've also changed the test to meet this criteria :)